virtualbox/machine.go

```diff
@@ -690,13 +690,13 @@ func (m *Machine) Modify() error {
 
 // AddNATPF adds a NAT port forarding rule to the n-th NIC with the given name.
 func (m *Machine) AddNATPF(n int, name string, rule driver.PFRule) error {
-	return vbm("controlvm", m.Name, fmt.Sprintf("natpf%d", n),
+	return vbm("modifyvm", m.Name, fmt.Sprintf("--natpf%d", n),
```

Comment:
The reason is that "controlvm" only works if the VM is already running. Is boot2docker-cli supposed to start the VM before performing this? If so, it doesn't do that on VBox 5.

Or did those commands work on VBox 4 without starting the VM first? I do not know.